fileconnector/fileconnector-1.0.0/org.wso2.carbon.connector/src/main/java/org/wso2/carbon/connector/FileExist.java

```diff
@@ -63,7 +63,7 @@ public void connect(MessageContext messageContext) throws ConnectException {
 		try {
 			isFileExist = isFileExist(fileLocation, filename, content);
 		} catch (FileSystemException e) {
-			handleException(e.getMessage(), messageContext);
+			handleException(e.getMessage(),e, messageContext);
```

Comment:
Why we are changing the style here. In earlier cases it's printing the error message.